src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+spring:
+  application:
+    name: book-rest-api
+  datasource:
+    url: "jdbc:postgresql://localhost:5432/bookrestapi"
+    username: "postgres"
```

Comment:
Create a placeholders to all fields for conection (username, password, schema name) and set up this placeholders in project configuration